tests/tConvert.cc

```diff
@@ -126,13 +126,15 @@ namespace casacore { namespace pyrap {
 BOOST_PYTHON_MODULE(_tConvert)
 {
   // Register the required converters.
-  casacore::pyrap::register_convert_excp();
-  casacore::pyrap::register_convert_basicdata();
-  casacore::pyrap::register_convert_casa_valueholder();
-  casacore::pyrap::register_convert_casa_record();
-  casacore::pyrap::register_convert_std_vector<casacore::uInt>();
-  casacore::pyrap::register_convert_std_vector<std::vector<casacore::uInt> >();
+  casacore::python::register_convert_excp();
+  casacore::python::register_convert_basicdata();
+  casacore::python::register_convert_casa_valueholder();
+  casacore::python::register_convert_casa_record();
+  casacore::python::register_convert_std_vector<bool>();
```

Comment:
I'm always wary of registering converters for purely standard library types, because it may affect operation of any other Boost.Python-using module that uses these types.

Reply:
so you think it is best to disable std_vector<bool> ? @gervandiepen what do you think? Or @tammojan

Reply:
I was thinking that `std::vector<casacore::uInt>` wasn't a problem and that it's only `std::vector<bool>` that is a potential issue (and which seems to be new in this PR). But actually casacore::uInt is (I assume) just a typedef, not a genuinely new type, so casacore is already doing this, and presumably disabling `std::vector<casacore::uInt>` would cause issues.